x/websocket/endpoint.go

```diff
@@ -160,8 +160,11 @@ func (c *gorillaConn) Read(buf []byte) (int, error) {
 	}
 	if err != nil {
 		var closeError *websocket.CloseError
-		if errors.As(err, &closeError) && closeError.Code == websocket.CloseNormalClosure {
-			return 0, io.EOF
+		if errors.As(err, &closeError) {
+			if websocket.IsCloseError(err, websocket.CloseNormalClosure) {
```

Comment:
No need to check if it's a close error twice. Also, the code forced me to lookup what the `websocket.IsCloseError` function does, so it affects readability as one more thing to know about.
```suggestion
			if closeError.Code == websocket.CloseNormalClosure {
```